doc/tutorials/13-charged_rod/13-charged_rod.ipynb

```diff
@@ -0,0 +1,536 @@
+    "rod_radius = 1.0\n",
+    "\n",
+    "# particle types \n",
+    "rod_type = 1\n",
+    "counterion_type = 2\n",
+    "\n",
+    "#ion-ion interaction\n",
+    "system.non_bonded_inter[counterion_type,counterion_type].wca.set_params(\n",
+    "      epsilon=wca_epsilon, sigma=ion_diameter)\n",
+    "\n",
+    "# ion-rod interaction\n",
+    "system.non_bonded_inter[counterion_type,rod_type].wca.set_params(\n",
+    "      epsilon=wca_epsilon, sigma=ion_diameter/2. + rod_radius)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def setup_rod_and_ions(system, ion_valency, counterion_type,\n",
+    "                 rod_line_charge, N_rod_beads, rod_type):\n",
+    "    \n",
+    "    # calculate charge of the single rod beads\n",
+    "    total_rod_charge = rod_charge_dens*system.box_l[0]\n",
+    "    rod_charge_per_bead = total_rod_charge/N_rod_beads\n",
+    "    \n",
+    "    # number of counterions\n",
+    "    N_ions = int(total_rod_charge/ion_valency)\n",
+    "    \n",
+    "    assert(abs((total_rod_charge-N_ions*ion_valency)/total_rod_charge)<1e-8)\n",
+    "    \n",
+    "    for idx in range(N_rod_beads):\n",
+    "        system.part.add(pos=[rod_length/2.,rod_length/2.,idx/N_rod_beads], \n",
+    "                        type=rod_type, q=rod_charge_per_bead, fix=3*[True])\n",
+    "\n",
+    "    for _ in range(N_ions):\n",
+    "        system.part.add(pos=np.random.random(3) * system.box_l, \n",
+    "                        type=counterion_type, q=-ion_valency)\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "counterion_valency = 2\n",
+    "rod_charge_dens = 1\n",
+    "\n",
+    "\n",
+    "\n",
+    "setup_rod_and_ions(system, counterion_valency, counterion_type,\n",
+    "             rod_charge_dens, N_rod_beads, rod_type)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "p3m_params = {'prefactor':kT*bjerrum_length,\n",
+    "              'accuracy':1e-3}\n",
+    "p3m = electrostatics.P3M(**p3m_params)\n",
+    "system.actors.add(p3m)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def remove_overlap(system, sd_params):   \n",
+    "    #remove overlap by steepest descent\n",
+    "    # Set up steepest descent integration\n",
+    "    system.integrator.set_steepest_descent(f_max=sd_params['f_max'],\n",
+    "                                           gamma=sd_params['damping'],\n",
+    "                                           max_displacement=sd_params['max_displacement'])\n",
+    "    \n",
+    "    # Initialize integrator to obtain initial forces\n",
+    "    system.integrator.run(0)\n",
+    "    maxforce = np.max(np.linalg.norm(system.part[:].f))\n",
```

Comment:
Here and in the loop below, `maxforce` contains the square root of the sum of squared forces. Did you mean to calculate the maximal force?
```suggestion
    "    maxforce = np.max(np.linalg.norm(system.part[:].f, axis=1))\n",
```